instructure/canvas-lms
# VeriCite: add encryption flag for submissions to specify encryption type

Test Plan:
1) Enable VeriCite in plugins. Use a real key and secret.
2) Create an assignment with VeriCite enabled
3) Submit to this assignment
4) Make sure report is generated (~10-15 mins)
